external_server/adapters/api/adapter.py

```diff
@@ -0,0 +1,356 @@
+import threading
+import sys
+
+sys.path.append("lib/fleet-protocol/protobuf/compiled/python")
+
+from InternalProtocol_pb2 import (  # type: ignore
+    Device as _Device,
+)
+from ExternalProtocol_pb2 import Status as _Status  # type: ignore
+from external_server.models.structures import (
+    Buffer,
+    DeviceIdentification,
+    DisconnectTypes,
+)
+from external_server.models.structures import (
+    GeneralErrorCode as _GeneralErrorCode,
+    ReturnCode,
+)
+from external_server.models.devices import device_repr
+from external_server.config import ModuleConfig
+from external_server.adapters.api.module_lib import (
+    empty_command_buffer as _empty_command_buffer,
+    empty_device_identification as _empty_device_identification,
+    ModuleLibrary as _ModuleLibrary,
+)
+from external_server.logs import CarLogger as _CarLogger
+
+
+_logger = _CarLogger(__name__)
+
+
+class APIClientAdapter:
+    """A wrapper around External server API functions (further API).
+
```

Comment:
All those sentences and the most important thing is missing: that this class provides connection with .so libs using ModuleLib

Reply:
Made it the first sentence of the dcostring